Lombiq.HelpfulExtensions.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -135,6 +136,57 @@ public static async Task TestFeatureCodeGenerationsAsync(this UITestContext cont
         });
     }
 
+    /// <summary>
+    /// Tests the Lombiq Helpful Extensions - Content Sets feature.
+    /// </summary>
+    public static async Task TestFeatureContentSetsAsync(this UITestContext context)
+    {
+        const string contentId0 = "contentsetexample000000000";
+        const string contentId2 = "contentsetexample000000002";
```

Comment:
Is this intentionally `contentId2` instead of `contentId1` ?

Reply:
Yes, it matches the ContentItemId. (these are the 1st and the 3rd items in this content set)